db/fixtures/miq_product_features.yml

```diff
@@ -4612,72 +4612,90 @@
     :feature_type: view
     :identifier: physical_infra_overview_view
 
-# Foreman
+# Configuration Management
 - :name: Configuration Management
   :description: Everything under Configuration Management
   :feature_type: node
-  :identifier: provider_foreman_explorer
+  :identifier: configuration_management
```

Comment:
What about renaming it to something more generic, what about `ems_configuration`? cc @Fryguy

Reply:
Is that the pattern we follow elsewhere? (like what do we do for cloud?)

Reply:
@skateman usually we try to keep same/similar to controller name or model name

Reply:
so for cloud/infra features, we went with ems_infra/ems_cloud controllers to match the model alias that is being used EmsInfra/EmsCloud. But in this case controller/class is ConfigurationManager hence i used prefix as configuration_manager for the feature identifiers as well as in other places in the UI code. We can create an alias EmsConfiguration to match the naming convention and rename controller/view files to follow the same to keep this simple and consistent

Reply:
I think you know that I am biased, I love consistency :laughing: there's ems_infra, ems_cloud and ems_container and if it 's possible, I'd like to keep this consistency as much as possible.

Reply:
@Fryguy what do you think about creating an alias `EmsConfiguration` for `ManageIQ::Providers::ConfigurationManager`

Reply:
That seems fine to create an alias for now